Automattic/jetpack
# Add/blaze add rest logs

This PR adds compatibility to Jetpack for the new /logs endpoint in the DSP

Fixes # 

## Proposed changes:
* Adds compatibility for log endpoint in DSP

### Other information:


## Does this pull request change what data or activity we track or use?

My PR doesn't add any tracking data

My PR adds *x* and *y*.
## Jetpack product discussion
n/a (internal slack convos)

## Testing instructions:
- Create a campaign with Blaze. 
- Make the widget to explode. Ask javiolmo89 in slack for instructions . this should create a test error and send a POST request to this new endpoint. The log should be correctly stored in `dsp_notes` database in DSP
